### PR TITLE
Updated readme with command to run the project

### PR DESCRIPTION
**Pull request by kieranirving**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ A staging environment is available at https://codidact.github.io/landing-page *(
 
 -   Ensure node and npm are installed locally [(nvm recommended)](https://github.com/nvm-sh/nvm).
 -   Run `npm install` to install project dependencies.
+-   Run `npm run start` to start the front-end. This is can be viewed by navigating to `localhost:3000` in your browser.
```
Comment (mattjbrent) [inline README.md]: remove` is` - `This can be viewed by navigating to ...`

**Comment by mattjbrent** [comment]
@kieranirving could you please rebase interactively and squash that 2nd commit to keep our git history clean, please and thank you

**Comment by kieranirving** [comment]
@mattjbrent this should be fixed now 😄

**Comment by ranolfi** [comment]
We'll need to rebase your changes and push this again before merging. Hit me up if you need help with that.

**Pull request merged**